WordPress/gutenberg
# [Mobile] Correct Color for RichText links in Dark Mode

**Fixes** https://github.com/wordpress-mobile/gutenberg-mobile/issues/2399

**Related PRs:**
`gutenberg-mobile` https://github.com/wordpress-mobile/gutenberg-mobile/pull/2404
`AztecEditor-iOS` https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1295
~`WordPress-Android` https://github.com/wordpress-mobile/WordPress-Android/pull/12226~
~`WordPress-iOS` https://github.com/wordpress-mobile/WordPress-iOS/pull/14351~



## Description

Moves `linkTextColor` to text element to help enforce the order of operations on the Native side.

For a full description please see the description in https://github.com/wordpress-mobile/gutenberg-mobile/pull/2404

## How has this been tested?



- Set your device too dark mode
- Create a post/page or open one that has links. 

Color should be using `blue-30` (`#5198d9`)

## Screenshots 
| Before | After |
| --- | --- |
|<kbr><img width=300 src="https://user-images.githubusercontent.com/3384451/85074845-38d0cd00-b18b-11ea-8502-2162fa6400cc.jpg"/></kbr> | <kbr><img width=300 src="https://user-images.githubusercontent.com/3384451/85075934-49824280-b18d-11ea-9c16-8702ecd93248.jpg"/></kbr> |

## Types of changes




BugFix

## Checklist:
